From: Ichthyostega <prg@ichthyostega.de>
Date: Thu, 17 May 2018 17:37:49 +0200
Subject: Fix Linux Version information

the main Linux Client Version string was fixed upstream meanwhile,
which is prerequisite for the display of "Gold Sheep"

However, we also fix some further settings in the build system
- disable the BETA_RELEASE flag, since this package is based on a release version
- init Autotools with the proper version string
---
 Common/clientversion.h | 2 +-
 configure.ac           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Common/clientversion.h b/Common/clientversion.h
index a9d7af3..2c666c1 100644
--- a/Common/clientversion.h
+++ b/Common/clientversion.h
@@ -33,7 +33,7 @@
 #define REDIRECT_SERVER				"community.sheepserver.net"
 #define REDIRECT_SERVER_FULL		"https://community.sheepserver.net"
 
-#define	BETA_RELEASE
+//#define	BETA_RELEASE
 
 
 #endif // CLIENTVERSION_H_INCLUDED
diff --git a/configure.ac b/configure.ac
index e9a562c..eb158d9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,7 +4,7 @@ AC_PREREQ(2.57)
 ifdef([m4_pattern_allow],
       [m4_pattern_allow([^AC_.*])])
 
-AC_INIT(electricsheep,2.7b33-svn,http://code.google.com/p/electricsheep/issues/list)
+AC_INIT(electricsheep,3.0.2,http://code.google.com/p/electricsheep/issues/list)
 AC_CANONICAL_HOST
 AC_CANONICAL_TARGET
 
